src/main/java/uc/doc/submitsolution/SubSDocDsRequestModel.java

```diff
@@ -2,24 +2,63 @@
 
 public class SubSDocDsRequestModel {
```

Comment:
JavaDoc should be added to each class to describe what the class does and what layer it belongs to (using the @layer tag).

Reply:
I agree! For the sake of consistency, take a look at the Login and Register Use cases! I believe they can be used as a guideline!